aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -156,54 +182,29 @@ void f8f8bf16_rowwise_impl(
                           // the Collective Builder
 
   // Implement rowwise scaling epilogue.
-  using XScale = cutlass::epilogue::fusion::Sm90ColBroadcast<
-      0,
-      TileShape,
-      DtypeScale,
-      cute::Stride<cute::Int<1>, cute::Int<0>, cute::Int<0>>>;
-
-  using WScale = cutlass::epilogue::fusion::Sm90RowBroadcast<
-      PONG ? 2 : 1,
-      TileShape,
-      DtypeScale,
-      cute::Stride<cute::Int<0>, cute::Int<1>, cute::Int<0>>>;
-
-  using Bias = cutlass::epilogue::fusion::Sm90RowBroadcast<
-      PONG ? 2 : 1,
-      TileShape,
-      DtypeBias,
-      cute::Stride<cute::Int<0>, cute::Int<1>, cute::Int<0>>>;
+  constexpr int ColBcastStages = 0;
```

Comment:
should this be ColA
?

Reply:
I was using `Bcast` as a shorthand for `Broadcast` but let me expand it to avoid confusion.

Reply:
ohhh my bad I though it was Cols B tensor